internal/searcher/searcher.go

```diff
@@ -15,9 +15,9 @@ import (
 
 func SearchAndPrint(w io.Writer, query string) error {
 	c := Client()
-	result, err := c.Search(query)
+	result, err := c.Search(query, "")
```

Comment:
I split them up mostly to void this ^ 

passing in empty args hurts readability (especially strings and booleans) because the code reader now needs to click in to the function to understand what params are. 

One pattern that avoids this but uses a lot of boilerplate is "WithOptions":

```golang
result, err := c.Search(query)
result, err := c.Search(query, WithVersion(version))
```